docs/tables/pagerduty_incident.md

```diff
@@ -0,0 +1,70 @@
+# Table: pagerduty_incident
+
+An incident represents a problem or an issue that needs to be addressed and resolved.
+
+## Examples
+
+### List unacknowledged incidents
+
+```sql
+select
+  incident_number,
+  summary,
+  urgency,
+  created_at,
+  assignments
+from
+  pagerduty_incident
+where
+  status = 'triggered';
+```
+
+### List unacknowledged incidents with high urgency
```

Comment:
For these examples, can you please also add `created_at` and mention the timeframe in the example desc?